cloud-ide.md

```diff
@@ -1 +1,267 @@
+  - [Step 1: Setting up your Account](#step-1-setting-up-your-account)
+  - [Step 2: Creating the "Stack"](#step-2-creating-the-stack)
+  - [Step 3: Creating the "Workspace"](#step-3-creating-the-workspace)
+- [Using Codenvy](#using-codenvy)
+  - [The Terminal](#the-terminal)
+  - [File Explorer Pane](#file-explorer-pane)
+  - [Edit File Pane](#edit-file-pane)
+- [Setting Up Projects](#setting-up-projects)
+  - [Step 1: Creating the App](#step-1-creating-the-app)
+  - [Step 2: Setting up the Database (Postgres)](#step-2-setting-up-the-database-postgres)
+  - [Step 3: Setting up the Run Command](#step-3-setting-up-the-run-command)
+- [Alternatives](#alternatives)
+
+The important terms we'll be using with Codenvy to understand before we get started: 
+
+ - **stacks** define your environment requirements (e.g. you will need ruby)
+ - **workspaces** are your environment (e.g. a created machine with ruby installed)
+ - **projects** are folders of code, the applications themselves (e.g. a ruby on rails web application you've built)
+
+## Setting up Codenvy
+
+To get your coding environment set up on Codenvy, we'll need to do three things: set up our account, create a <a href="#stacks">stack</a> with all of the software we'll need, and then create a <a href="#workspaces">workspace</a>. Once the environment is set up, you'll want to [set up a project](#setting-up-projects).
+
+Follow these steps to get the environment set up.
+
+### Step 1: Setting up your Account
+
+1. Navigate to <https://codenvy.io/site/login> and click **Create One** to set up an account. 
+
+   <img width="350" alt="Login Screen" src="images/codenvy/account-setup/1.png" />
+
+2. Enter your email address and a username, and then click **Create Account**.
+
+   <img width="350" alt="Create Account Screen" src="images/codenvy/account-setup/2.png" />
+
+3. Close the confirmation page.
+
+   <img width="350" alt="Confirmation Screen" src="images/codenvy/account-setup/3.png" />
+
+4. Click the **Verify Email** button in the email sent to the address you entered in step 2.
+
+   <img width="350" alt="Email Verification" src="images/codenvy/account-setup/4.png" />
+
+5. Finish setting up your account by entering the requested information. Click **Get Started**.
+
+   <img width="350" alt="Account Setup" src="images/codenvy/account-setup/5.png" />
+
+### Step 2: Creating the "Stack"
+
+<span id="stacks">**Stacks**</span> define what programming languages, programs, and commands should be available on the "machine" your workspace is running on. Setting up a stack is the equivalent of making a list of software you need to install on your laptop.
+
+Follow these steps to set up the stack we'll be using:
+
+1. Click **Stacks** in the left navigation, and then click **Build Stack From Recipe**.
+
+   <img width="350" alt="Create Stack Step 1" src="images/codenvy/account-setup/6.png" />
+
+2. Paste `FROM tteltrab/rr-pg-for-codenvy` into the box, and click **OK**.
```

Comment:
I think we might want to make this a little more explicit what the steps are.

2. Find the the textbox that is presented for the x field and enter the following code in there:

```
FROM tteltrab/rr-pg-for-codenvy
```

Then click the `OK` button.

Reason being, when I first read this I wasn't sure if there was a separate thing that I should copy and paste from (as opposed to this specific value).